quicklink.admin.inc

```diff
@@ -162,8 +162,8 @@ function quicklink_settings($form, $form_state) {
  * {@inheritdoc}
  */
 function quicklink_settings_submit(&$form, $form_state) {
-
   $config = config('quicklink.settings');
+  $debug_mode = config_get('quicklink.settings', 'enable_debug_mode');
```

Comment:
Already have a Config object, so use it: `$debug_mode = $config->get('enable_debug_mode');`